prow/config.yaml

```diff
@@ -933,6 +933,78 @@ periodics:
         path: /mnt/disks/ssd0
       name: cache-ssd
 
+- name: ci-kubernetes-e2e-gce-ubuntu-node
+  interval: 30m
+  spec:
+    containers:
+    - args:
+      - --timeout=50
+      - --bare
+      - --gce-ssh=/etc/ssh-key-secret/ssh-private
+      - --gce-pub=/etc/ssh-key-secret/ssh-public
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
+        value: prow
+      image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170418-c08e1094
+      volumeMounts:
+      - mountPath: /etc/service-account
+        name: service
+        readOnly: true
+      - mountPath: /etc/ssh-key-secret
+        name: ssh
+        readOnly: true
+      - mountPath: /root/.cache
+        name: cache-ssd
+    volumes:
+    - name: service
+      secret:
+        secretName: service-account
+    - name: ssh
+      secret:
+        defaultMode: 256
+        secretName: ssh-key-secret
+    - hostPath:
+        path: /mnt/disks/ssd0
+      name: cache-ssd
+
+- name: ci-kubernetes-e2e-gce-ubuntu-node-serial
+  interval: 30m
+  spec:
+    containers:
+    - args:
+      - --timeout=180
+      - --bare
+      - --gce-ssh=/etc/ssh-key-secret/ssh-private
+      - --gce-pub=/etc/ssh-key-secret/ssh-public
```

Comment:
```
W0502 05:51:49.351] Run: ('docker', 'pull', 'gcr.io/k8s-testimages/kubekins-node:1.7-latest')
W0502 05:51:49.406] Cannot connect to the Docker daemon. Is the docker daemon running on this host?
W0502 05:51:49.407] Traceback (most recent call last):
W0502 05:51:49.407]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_kubelet.py", line 129, in <module>
W0502 05:51:49.408]     ARGS.service_account,
W0502 05:51:49.408]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_kubelet.py", line 76, in main
W0502 05:51:49.408]     check('docker', 'pull', img)  # Update image if changed
W0502 05:51:49.409]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_kubelet.py", line 45, in check
W0502 05:51:49.409]     subprocess.check_call(cmd)
W0502 05:51:49.409]   File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
W0502 05:51:49.409]     raise CalledProcessError(retcode, cmd)
W0502 05:51:49.410] subprocess.CalledProcessError: Command '('docker', 'pull', 'gcr.io/k8s-testimages/kubekins-node:1.7-latest')' returned non-zero exit status 1
E0502 05:51:49.412] Build failed
I0502 05:51:49.412] process 261 exited with code 1 after 0.0m
E0502 05:51:49.412] FAIL: ci-kubernetes-node-kubelet
```

https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_kubelet.py#L77-L94 tried to pull an image. Is docker available in prow-e2e?